tests/scripts/thread-cert/thread_cert.py

```diff
@@ -500,9 +520,6 @@ def _parse_params(self, params: Optional[dict]) -> dict:
             assert params.get('version', '') == '', params
             params['version'] = ''
 
-        # set default backbone network for bbr/otbr/host if not specified
-        params.setdefault('backbone_network', config.BACKBONE_DOCKER_NETWORK_NAME)
```

Comment:
keeping this line to ensure it will never be None, which can simply the _construct_backbone_network_name

Reply:
The `backbone_network_id` is `None` for non otbr/host nodes by design in this PR. It can be strange (though no harm)  to assign a default backbone network id for a non-BR node. Any thoughts?